ruby/.rubocop.yml

```diff
@@ -0,0 +1,16 @@
+# Most of these are disabling existing cops, primarily
```

Comment:
Fair point. I guess I've been enjoying myself too much!

I personally think the code (ignoring all the meta) is pretty solid, though, and I personally feel the structure is very readable and forward-refactorable / portable.

FYI: Rubocop often annoys the XXX out of me...